src/chigame/users/urls.py

```diff
@@ -2,9 +2,12 @@
 
 from chigame.users.views import user_detail_view, user_redirect_view, user_update_view
 
+from . import views
```

Comment:
you can just add "user_list" to the import line above

Reply:
Thanks for the simplification, I added it on line 3 instead of creating a new line.